### PR TITLE
swipe left to open menu on phone

### PR DESCRIPTION
**Pull request by hcaumeil**

### CODE REVIEW

File: web/public/js/ui/top-bar.js [hunk 1]
```diff
@@ -1,17 +1,54 @@
...
+                                                                                    
+                if ( Math.abs( xDiff ) > Math.abs( yDiff ) ) {/*most significant*/
+                        if ( xDiff < 0 ) { // swipe left
+                            document.getElementsByClassName("sidebar icon").item(null).click()
+                        }                       
+                }
+
+                xDown = null;
+                yDown = null;                                             
+            };
```
Comment (alyrow) [inline web/public/js/ui/top-bar.js]: Can be move outside the forEach at line 12

File: web/public/js/ui/top-bar.js [hunk 2]
```diff
@@ -1,17 +1,54 @@
...
+            function handleTouchMove(evt) {
+                if ( ! xDown || ! yDown ) {
+                    return;
+                }
+                            
+                var xUp = evt.touches[0].clientX;                                    
+                var yUp = evt.touches[0].clientY;
+            
+                var xDiff = xDown - xUp;
+                var yDiff = yDown - yUp;
```
Comment (alyrow) [inline web/public/js/ui/top-bar.js]: Usage of `let` is better

File: web/public/js/ui/top-bar.js (hunk 1 above)
Comment (alyrow) [inline web/public/js/ui/top-bar.js]: Move functions before `document.addEventListner` and transform them in expression (I find that more clear in this context) like this:

```js
const hello = function(name) {
   console.log("Hello " + name);
}
```

**Review by alyrow — CHANGES_REQUESTED** [review]

**Comment by hcaumeil** [comment]
Changes have been done

**Review by alyrow — APPROVED** [review]

**Pull request merged**